src/dashboard/apps/consent/utils.py

```diff
@@ -0,0 +1,21 @@
+"""Dashboard consent app utils."""
+
+from datetime import datetime, timedelta, timezone
+
+
+def calculate_consent_end_date(days: int | None = None) -> datetime:
```

Comment:
Ensure that you have the appropriate configurable default when the utility is called without (kw)args (like in a model field `default` :wink:):

```suggestion
def calculate_consent_end_date(days: int | None = CONSENT_NUMBER_DAYS_END_DATE) -> datetime:
```